README.md

```diff
@@ -4,6 +4,45 @@ An extensible arbitrage bot for trading against valence auctions and Astroport a
 
 ## Installation
 
+### Nix
+
+Use [Nix](https://github.com/NixOS/nix) to automatically install all required dependencies, or [manually](#manual) install dependencies.
+
+Nix can be installed through the following steps on MacOS and Linux:
+
+#### MacOS
+
+```sh
+curl -L https://nixos.org/nix/install | sh
+```
+
+#### Linux
+
+```sh
+curl -L https://nixos.org/nix/install | sh -s -- --daemon
+```
+
+After installing Nix, enter the abritrage bot dev environment by executing
+
+```sh
+nix develop
```

Comment:
I had to use the following `nix --extra-experimental-features nix-command --extra-experimental-features flakes` on a fresh nix shell install .. don't know if that's normal.

Reply:
Good point. I will amend the REAMDE.